source/views/building-hours/stateful-list.js

```diff
@@ -48,16 +74,15 @@ export class BuildingHoursView extends React.Component<Props, State> {
     loading: false,
     // now: moment.tz('Wed 7:25pm', 'ddd h:mma', null, CENTRAL_TZ),
     now: moment.tz(CENTRAL_TZ),
-    buildings: groupBuildings(defaultData.data),
+    buildings: loadAllBuildings(defaultData.data, this.props.favoriteBuildings),
     intervalId: 0,
   }
 
   componentWillMount() {
     this.fetchData()
-
     // This updates the screen every ten seconds, so that the building
     // info statuses are updated without needing to leave and come back.
-    this.setState({intervalId: setInterval(this.updateTime, 1000)})
+    this.setState({intervalId: setInterval(this.updateTime, 10000)})
```

Comment:
This change should be reverted, please.

Reply:
I think he was going off of the comment that says "ten seconds" so let's change the comment if we're going to leave the code at one second.

Reply:
Yes, we should change the comment.